### PR TITLE
Update method name for calling faucet contract

### PR DESCRIPTION
**Pull request by rweald**
registerAccount was causing some confusion, and what the method actually
does is allow caller to claim RLY tokens. This updates the method name,
but leaves in place the older method with a deprecation warning.

### CODE REVIEW

**Comment by rweald** [comment]
I made the change backwards compatible. The old method is still in there, it just logs an error to console telling people to update their method call, and all the actual logic has been moved to new method.

File: example/src/AccountOverviewScreen.tsx [hunk 1]
```diff
@@ -111,7 +111,7 @@ export const AccountOverviewScreen = (props: { rlyAccount: string }) => {
             <View style={styles.alignMiddle}>
               <BodyText>Register My Account</BodyText>
             </View>
-            <Button onPress={registerAccount} title="Register" />
+            <Button onPress={claimRlyTokens} title="Register" />
```
Comment (jonorally) [inline example/src/AccountOverviewScreen.tsx:114]: Should we make this "Claim Tokens" also or does that make our sign up page look like a spam website lol

File: src/__tests__/integration/transaction.test.ts [hunk 2]
```diff
@@ -43,6 +43,19 @@ testOnlyRunInCIFullSuite(
...
+    expect(oldBal).toEqual(0);
+    expect(txHash).toMatch(/^0x/);
+    expect(newBal).toEqual(10);
+  },
+  30000
+);
+
 test('claim local', async () => {
   const account = await getWallet();
   const oldBal = await RlyDummyNetwork.getBalance();
```
Comment (jonorally) [inline src/__tests__/integration/transaction.test.ts:61]: Can do this in a later PR but there's another registerAccount down here too

**Review by jonorally — APPROVED** [review]

File: example/src/AccountOverviewScreen.tsx (hunk 1 above)
Comment (rweald) [reply example/src/AccountOverviewScreen.tsx:114]: That's a good point. Makes sense from a consistency standpoint

**Pull request merged**